src/generator/AutoRest.AzureResourceSchema/JsonSchema.cs

```diff
@@ -13,6 +13,7 @@ namespace AutoRest.AzureResourceSchema
     /// </summary>
     public class JsonSchema
     {
+        private string resourceType;
```

Comment:
No, I removed it because relying on an entry with the name "type" is ambiguous (see my other comment regarding setting this value).